src/components/Registration/BusinessNumber.vue

```diff
@@ -19,8 +19,10 @@
           If you have an existing business number, enter it below and we will contact
           Canada Revenue Agency and ask them to link it to this registration.
         </p>
-        <!-- NB: this component is only used for GPs -->
-        <HelpBusinessNumber class="mt-4" isTypePartnership=true />
+        <HelpBusinessNumber class="mt-4"
+          :isTypePartnership="isTypePartnership"
+          :isTypeSoleProp="!isTypePartnership"
```

Comment:
I think it would be better to use the other getter, in case there is ever a 3rd type. Plus this will make the code nice and symmetric :)

Reply:
Looks good, thanks.